app/src/main/java/com/epfl/beatlink/ui/library/EditPlaylist.kt

```diff
@@ -148,7 +177,7 @@ fun EditPlaylistScreen(
                           nbTracks = selectedPlaylistState.nbTracks)
                   playlistViewModel.updatePlaylist(updatedPlaylist)
                   playlistViewModel.selectPlaylist(updatedPlaylist)
-                  navigationActions.navigateTo(PLAYLIST_OVERVIEW)
+                  navigationActions.navigateToAndClearBackStack(PLAYLIST_OVERVIEW, 1)
```

Comment:
I added this because otherwise if we click on the back arrow it returns to the EDIT SCREEN instead of just going back to the list of playlists screen.